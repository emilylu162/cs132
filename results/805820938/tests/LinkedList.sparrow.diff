9,11d8
< (10 + 0)
< (6 + 0)
< (2 + 0)
13,15d9
< (10 + 0)
< (6 + 0)
< (2 + 0)
