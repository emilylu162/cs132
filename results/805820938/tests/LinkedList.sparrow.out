25
10000000
39
25
10000000
22
39
25
(10 + 0)
(6 + 0)
(2 + 0)
1
(10 + 0)
(6 + 0)
(2 + 0)
0
10000000
28
22
39
25
2220000
-555
-555
28
22
25
33300000
22
25
44440000
0
